.gitignore

```diff
@@ -0,0 +1,6 @@
+# dev
+/companies
+/teams
+/projects
+/tmp
+/tmp_out
```

Comment:
Is that local dev output?

Reply:
yes

Reply:
used to add files with '-A' flag and this prevents from checking in temp files when running the commands from README.md